### PR TITLE
Adds changelog for WD6

### PR DESCRIPTION
**Pull request by LJWatson**
Changes between WD5 and WD6.

### CODE REVIEW

File: sections/changes.include [hunk 1]
```diff
@@ -5,6 +5,29 @@
...
+  <dt><a href="https://github.com/w3c/html/commit/c5b7996e390e29bee7101a33604fcba5c3d4dcc0">Clarified restrictions on children of p elements</a></dt>
+  <dd>Clarifies that child elements of the <{p}> element cannot be inline blocks, inline tables, or floated and positioned block-level elements.</dd>
+  <dt><a href="https://github.com/w3c/html/commit/c1cbc95ed3d911b23e3607a1803cfd181fbbc802">Make HTML4/xHTML1 strict doctypes non-conforming</a></dt>
+  <dd>Disallows the use of the HTML4 and xHTML 1.1 strict doctypes.</dd>
+  <dt><a href="https://github.com/w3c/html/commit/b75ed699d0db74790ce9d61bad711cc9ecd5e053">Clarify where to propose rel extensions</a></dt>
+  <dd>References the correct part of the Microformats wiki, and adds a note to indicate proposals can also be filed against this HTML specification.</dd>
+  <dt><a href="https://github.com/w3c/html/commit/49073d89a7829dc3dbbfc3802f38bf9845902117">Remove references to Polyglot Markup</a></dt>
+  <dd>Removes references to the Polyglot Markup specification (now published as a W3C Note).</dd>
+  </dl>
+
```
Comment (chaals) [inline sections/changes.include:30]: There should be a fixed version link for the *following* reference to WD5

File: sections/changes.include [hunk 2]
```diff
@@ -5,6 +5,29 @@
 
   Full details of all changes since 12 January 2016 are available from the <a href="https://github.com/w3c/html/commits/master">commit log</a> of the <a href="https://github.com/w3c/html/">w3c/html github repository</a>, including various editorial and linking fixes.
 
+  <h3 id="changes-wd5">Changes between <a href="https://www.w3.org/TR/html52/">Working Draft 6</a> and <a href="https://www.w3.org/TR/2017/WD-html52-20170228/">Working Draft 5</a></h3>
+  
+  <dl>
```
Comment (chaals) [inline sections/changes.include:10]: Missing changes to menu/menuitem:

[Make menu and menuitem match reality](https://github.com/w3c/html/commit/0b959a586e54fe4145b69303ebb11755cea26c09)
Substantive changes:
*remove `HTMLElement.contextmenu` IDL attribute
*remove the `radio` and `checkbox` types for menus
*Make `menuitem` have content, as per implementation.
*A `menuitem` makes a command if it is labeled
*change terminology from "popup menu" state to "context menu" state.

File: sections/changes.include [hunk 3]
```diff
@@ -5,6 +5,29 @@
...
+  <dt><a href="https://github.com/w3c/html/commit/c5b7996e390e29bee7101a33604fcba5c3d4dcc0">Clarified restrictions on children of p elements</a></dt>
+  <dd>Clarifies that child elements of the <{p}> element cannot be inline blocks, inline tables, or floated and positioned block-level elements.</dd>
+  <dt><a href="https://github.com/w3c/html/commit/c1cbc95ed3d911b23e3607a1803cfd181fbbc802">Make HTML4/xHTML1 strict doctypes non-conforming</a></dt>
+  <dd>Disallows the use of the HTML4 and xHTML 1.1 strict doctypes.</dd>
+  <dt><a href="https://github.com/w3c/html/commit/b75ed699d0db74790ce9d61bad711cc9ecd5e053">Clarify where to propose rel extensions</a></dt>
+  <dd>References the correct part of the Microformats wiki, and adds a note to indicate proposals can also be filed against this HTML specification.</dd>
+  <dt><a href="https://github.com/w3c/html/commit/49073d89a7829dc3dbbfc3802f38bf9845902117">Remove references to Polyglot Markup</a></dt>
+  <dd>Removes references to the Polyglot Markup specification (now published as a W3C Note).</dd>
+  </dl>
```
Comment (chaals) [inline sections/changes.include:29]: Missing [allow role="none" on `img`](https://github.com/w3c/html/commit/0b959a586e54fe4145b69303ebb11755cea26c09)

For `img alt=""` only

**Review by chaals — CHANGES_REQUESTED** [review]

**Review by travisleithead — APPROVED** [review]

**Comment by chaals** [comment]
So there is a changelog in the Working Draft... meaning this PR can just be closed, no? @LJWatson

**Pull request closed without merging**